src/constants.js

```diff
@@ -44,4 +44,13 @@ export const SORT_TYPES = {
   DESCENDING: 'descending',
 };
 
+export const LOCATION_LIBRARY_ID_KEY = 'locationLibraryId';
+export const LOCATION_CAMPUS_ID_KEY = 'locationCampusId';
+export const LOCATION_INSTITUTION_ID_KEY = 'locationInstitutionId';
+
+export const INSTITUTION_ID_LIBRARIES = 'institutionIdLibraries';
+export const INSTITUTION_ID_CAMPUS = 'institutionIdCampuses';
+export const CAMPUS_ID_LIBRARIES = 'campusIdLibraries';
```

Comment:
It's not obvious what these values are for, especially since they are defined distant from where they are used (I think they are session storage keys? Do you think there should be some sort of namespacing or common pattern/function for such?). Please add a comment explaining their purpose.

Reply:
Yes sure. Last part of INSTITUTION_ID_LIBRARIES means tab of app.